sources/data-node/api/v1/trading_data.proto

```diff
@@ -1219,25 +1222,39 @@ message OracleDataBySpecResponse {
   repeated oracles.v1.OracleData oracle_data = 1;
 }
 
-message ObserveRewardDetailsRequest {
+message ObserveRewardsRequest {
   string asset_id = 1;
-  string party = 2; 
+  string party = 2;
 }
 
-message ObserveRewardDetailsResponse {
-  vega.RewardDetails reward_details = 1;
+message ObserveRewardsResponse {
+  vega.Reward reward = 1;
 }
 
 // A request to get reward details for a party
-message GetRewardDetailsRequest {
+message GetRewardSummariesRequest {
   string party_id = 1 [(validator.field) = {string_not_empty : true }];
+  string asset_id = 2;
 }
 
 // The return message with reward details in for a single party
-message GetRewardDetailsResponse {
-  repeated vega.RewardPerAssetDetail reward_details = 1;
+message GetRewardSummariesResponse {
+  repeated vega.RewardSummary summaries = 1;
 }
```

Comment:
Why the names changes here? not so import  but this will be a breaking change.

Reply:
They just seemed more logical to me, and I found it quite confusing when I was trying to figure out what was going on initially; e.g. the RewardDetails contained aggregate information about rewards, where as the information about actual rewards were in RewardPerAssetDetails.

 In my opinion words like 'Info' 'Details' 'Data' generally don't add a lot of information to a name.